### PR TITLE
GPU support, BN, epochs update

### PR DESCRIPTION
**Pull request by HMurawski**
Updated training process: added GPU support, batch normalization, and increased epochs

### CODE REVIEW

**Comment by lzdunowski** [comment]
Changes from here are included in https://github.com/lzdunowski/torch-testing/pull/3.
I think we can close this PR and focus on second one.
